backend/src/module/core/__init__.py

```diff
@@ -1 +0,0 @@
-from .program import Program
```

Comment:
it seems used in `backend/src/module/api/program.py`

```python
from module.core import Program
```

there appear to be similar issues with other changes in `__init__.py` as well